app/src/userpages/components/ShareDialog/index.jsx

```diff
@@ -37,7 +37,7 @@ export const ShareDialog = (props: Props) => {
     const isMounted = useIsMounted()
 
     const {
-        allowEmbed,
+        allowEmbed = true,
```

Comment:
If you turn this on by default, the stream & dashboard share dialog will have the embed tab as well. I think for now leave this off by default and just enable the flag on the above modals?

Reply:
Ah you're right, my bad.

Reply:
Fixed in 34e51d1